components/src/core/components/Input/Select/SelectInputButton.vue

```diff
@@ -6,18 +6,19 @@
     <oxd-button
       class="dropdown-btn"
       :class="dropdownButtonClasses"
-      :label="buttonData.label"
-      :iconName="buttonData.iconName"
+      :label="!(modelValue && modelValue.id > -1) ? buttonData.label : null"
```

Comment:
Check that Select Input button is used by other devs

Reply:
This is not used by other devs yet